src/ppx/graphics_util.cpp

```diff
@@ -672,35 +715,40 @@ Result CreateImageFromCompressedImage(
         SCOPED_DESTROYER.AddObject(targetImage);
     }
 
-    // Copy info
-    grfx::BufferToImageCopyInfo copyInfo = {};
-    copyInfo.srcBuffer.imageWidth        = imageWidth;
-    copyInfo.srcBuffer.imageHeight       = imageHeight;
-    copyInfo.srcBuffer.imageRowStride    = rowStride;
-    copyInfo.srcBuffer.footprintOffset   = 0;
-    copyInfo.srcBuffer.footprintWidth    = imageWidth;
-    copyInfo.srcBuffer.footprintHeight   = imageHeight;
-    copyInfo.srcBuffer.footprintDepth    = 1;
-    copyInfo.dstImage.mipLevel           = 0;
-    copyInfo.dstImage.arrayLayer         = 0;
-    copyInfo.dstImage.arrayLayerCount    = 1;
-    copyInfo.dstImage.x                  = 0;
-    copyInfo.dstImage.y                  = 0;
-    copyInfo.dstImage.z                  = 0;
-    copyInfo.dstImage.width              = imageWidth;
-    copyInfo.dstImage.height             = imageHeight;
-    copyInfo.dstImage.depth              = 1;
+    for (gli::texture::size_type level = 0; level < image.levels(); level++) {
```

Comment:
The `ImageOptions` parameter can specify the number of mips to load - we should honor that.

Reply:
Ah, thanks.